make/proto.mk

```diff
@@ -24,12 +24,19 @@ proto.gen.doc:
 
 .PHONY: proto.gen.init
 proto.gen.init:
-	 go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
-	 go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
 
 .PHONY: proto.gen.code
 proto.gen.code:
 	$(DOCKER) build -t layotto/protoc $(ROOT_DIR)/docker/proto && \
 	$(DOCKER) run --rm \
 		-v  $(ROOT_DIR)/spec/proto/runtime/v1:/api/proto \
 		layotto/protoc
+
+.PHONY: proto.comments
+proto.comments:
+	sudo true
+	curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh | sudo -u $$USER bash
```

Comment:
What happened if developers already have Homebrew installed?

Reply:
> What happened if developers already have Homebrew installed?

It will check Homebrew is newest version. If not, it will update.

Reply:
Can we install buf binary according to the OS? As far as I know, homebrew is just a package manager in MacOS, what if user is using other OS like centOS/Unbuntu.

Reply:
> Can we install buf binary according to the OS? As far as I know, homebrew is just a package manager in MacOS, what if user is using other OS like centOS/Unbuntu.

It can work if user is using other OS like centOS/Unbuntu. The concern is that homebrew will take a time to install at the first time.